lcd/config_downloader.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+import uuid
+import requests
+
+class ConfigDownloader:
+    def __init__(self, config_path, remote_url):
+        self.config_path = config_path
+        self.remote_url  = remote_url
+        self.api_key      = uuid.uuid4()
+
+    def run(self):
+        print "Setting up a new account"
+        print "please make sure an account with API key %s is configured" % self.api_key
```

Comment:
I think you don't have to worry about the prompting and what not. I can take care of that in the lcd script. Ideally since this will be used as a module it would be good to have the functions in form.
`def generate_api_key()`
`def download_config(api_key)`
`def check_config_exists()`
This way the can easily be used as a module.